src/Text/Pandoc/ImageSize.hs

```diff
@@ -100,6 +102,8 @@ imageType img = case B.take 4 img of
                      "\xff\xd8\xff\xe0" -> return Jpeg  -- JFIF
                      "\xff\xd8\xff\xe1" -> return Jpeg  -- Exif
                      "%PDF"             -> return Pdf
+                     "<?xm"             -> return Svg -- this is more of a guess
```

Comment:
In this case couldn't we have a further test to see if `<svg` is an infix of the bytestring?  Or scan to the next `<` and verify that it's followed by `svg`?